neuro_san/http_sidecar/handlers/base_request_handler.py

```diff
@@ -121,35 +119,15 @@ def get_agent_grpc_session(self,
                 agent_name=agent_name)
         return grpc_session
 
-    def get_concierge_grpc_session(self, metadata: Dict[str, Any]) -> ConciergeSession:
-        """
-        Build gRPC session to talk to "concierge" service
-        :return: ConciergeSession to use
-        """
-        grpc_session: ConciergeSession = \
-            GrpcConciergeSession(
-                host="localhost",
-                port=self.port,
-                metadata=metadata)
-        return grpc_session
-
-    async def update_agents(self, metadata: Dict[str, Any]) -> bool:
+    async def update_agents(self) -> bool:
```

Comment:
How does the metadata get in here then?

Reply:
FWIW: CodedTools can use loggers however they want and they expect their metadata to be set up already in the logging infrastructure. 1C guys in particular are dependent on this.